src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -3628,21 +3628,25 @@ namespace HeatBalanceManager {
         Array1D<Real64> SCCenter(2);        // Center of glass shading coefficient for glazing system
         Array1D<Real64> SHGCCenter(2);      // Center of glass solar heat gain coefficient for glazing system
         Array1D<Real64> TVisCenter(2);      // Center of glass visible transmittance for glazing system
-        Array1D<Real64> Tsol(11);           // Solar transmittance vs incidence angle; diffuse trans.
-        Array2D<Real64> AbsSol(11, 5);      // Solar absorptance vs inc. angle in each glass layer
-        Array1D<Real64> Rfsol(11);          // Front solar reflectance vs inc. angle
-        Array1D<Real64> Rbsol(11);          // Back solar reflectance vs inc. angle
-        Array1D<Real64> Tvis(11);           // Visible transmittance vs inc. angle
-        Array1D<Real64> Rfvis(11);          // Front visible reflectance vs inc. angle
-        Array1D<Real64> Rbvis(11);          // Back visible reflectance vs inc. angle
-        Array1D<Real64> CosPhiIndepVar(10); // Cosine of incidence angle from 0 to 90 deg in 10 deg increments
-        int IPhi;                           // Incidence angle counter
-        Real64 Phi;                         // Incidence angle (deg)
-        Array1D<Real64> CosPhi(10);         // Cosine of incidence angle
-        Array1D<Real64> tsolFit(10);        // Fitted solar transmittance vs incidence angle
-        Array1D<Real64> tvisFit(10);        // Fitted visible transmittance vs incidence angle
-        Array1D<Real64> rfsolFit(10);       // Fitted solar front reflectance vs incidence angle
-        Array2D<Real64> solabsFit(5, 10);   // Fitted solar absorptance vs incidence angle for each glass layer
+        Array1D<Real64> TsolTemp(Window::numPhis+1);           // Solar transmittance vs incidence angle; diffuse trans.
```

Comment:
Need temporary `Array1D`'s in order to read from the W5 input file. Can probably move these to local variables now that I think about it.